src/calculateRentalCost.js

```diff
@@ -29,6 +29,20 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const rentPrice = 40;
+  const maxDaysDiscount = 7;
+  const minDaysDiscount = 3;
+  const withoutDiscount = 1;
+  const maxDiscount = 50;
+  const minDiscount = 20;
+
+  if (days >= maxDaysDiscount) {
+    return (days * rentPrice) - maxDiscount;
+  } else if (days >= minDaysDiscount) {
+    return (days * rentPrice) - minDiscount;
+  } else if (days >= withoutDiscount) {
+    return days * rentPrice;
+  }
```

Comment:
`else` is useless with `return` inside `if` block.
Also, you calculate `days * rentPrice` three times. Use variable for it.
```suggestion
  if (days >= maxDaysDiscount) {
    return (days * rentPrice) - maxDiscount;
  } 
  
  if (days >= minDaysDiscount) {
    return (days * rentPrice) - minDiscount;
  }
  
  return days * rentPrice;
```